iocore/net/quic/QUICStream.cc

```diff
@@ -947,3 +983,83 @@ QUICCryptoStream::_records_crypto_frame(QUICEncryptionLevel level, const QUICCry
   crypto_frame_info->block           = frame.data();
   this->_records_frame(frame.id(), std::move(info));
 }
+
+QUICOffset
+QUICCryptoStream::largest_offset_received() const
+{
+  // TODO:
+  ink_assert(!"unsupported");
+  return 0;
+}
+
+QUICOffset
+QUICCryptoStream::largest_offset_sent() const
+{
+  // TODO
+  ink_assert(!"unsupported");
+  return 0;
+}
+
+void
+QUICCryptoStream::stop_sending(QUICStreamErrorUPtr error)
+{
+  // TODO
+  ink_assert(!"unsupported");
+  return;
+}
+
+void
+QUICCryptoStream::reset(QUICStreamErrorUPtr error)
+{
+  // TODO
+  ink_assert(!"unsupported");
+  return;
+}
+
+void
+QUICCryptoStream::on_eos()
+{
+  return;
+}
+
+void
+QUICCryptoStream::on_read()
+{
+  return;
+}
+
+// Implement VConnection Interface.
+VIO *
+QUICCryptoStream::do_io_read(Continuation *c, int64_t nbytes, MIOBuffer *buf)
```

Comment:
Should CryptoStream support these VConnection interface? I'm fine with having these if we want to use them, but we could define these only for other streams (Bidi and Uni but not Crypto).

@masaori335 any thoughts?

Reply:
I'm not big fun of using VConnection Interface for CryptoStream. Recalling our implementation before draft-13 support (using stream 0 for handshake), we had bunch of troubles like timing issue comes from using VConnections.
As far as I understand, handshake doesn't need features which provided by VConnection - e.g. read/write event callback.

Reply:
Will remove it .